transformations/chinese_butter_fingers_perturbation/transformation.py

```diff
@@ -0,0 +1,227 @@
+import itertools
+import random
+import unidecode as u
+import json
+import os
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+import shutil
+import jieba
+import pypinyin
+
+
+def chinese_butter_finger(text,
+                          chinese_character_database,
+                          common_chinese_character_database,
+                          chinese_words_database,
+                          prob,
```

Comment:
Hi Maxime,

I kept this as `prob` to keep it consistent with the existing English `butter_fingers_perturbation` since I thought it would be confusing to the user if one transformation used `prob `and another used `replacement_prob` - maybe it would be as a follow-up if we standardize the arguments across different transformations once all the accepted transformations are known?

Reply:
ok, fine by me :)